bin/resources/GameIndex.yaml

```diff
@@ -24381,13 +24381,13 @@ SLPM-65767:
   name: "NBA Starting Five 2005"
   region: "NTSC-J"
 SLPM-65768:
-  name: "Beatmania IIDX - 8th Style"
+  name: "beatmania IIDX 8th style"
   region: "NTSC-J"
 SLPM-65769:
-  name: "Pop'n Music 10 [with Pop'n Controller 2]"
+  name: "pop'n music 10 [Controller Set]"
```

Comment:
The bundle as a whole is listed as SLPM-65769 and in the Playstation database as "pop'n music Controller Set", but the actual game inside is labeled SLPM-65769 and says pop'n music 10.

https://www.jp.playstation.com/software/title/slpm65769.html

https://twitter.com/mimi_hisakaki/status/1405144350141546497